lib/Test/Nginx/Util.pm

```diff
@@ -636,6 +640,38 @@ sub error_log_data () {
     return \@lines;
 }
 
+sub check_prevblock_error_log($) {
```

Comment:
Also, I don't think this function name is descriptive enough. You probably want to be more explicit about `shutdown_error_log` here.